packages/usa-banner/src/usa-banner.twig

```diff
@@ -1,8 +1,8 @@
 {% set lock %}
   <span class="icon-lock">
-    <svg xmlns="http://www.w3.org/2000/svg" width="52" height="64" viewBox="0 0 52 64" class="usa-banner__lock-image" role="img" aria-labelledby="banner-lock-title banner-lock-description" focusable="false">
+    <svg xmlns="http://www.w3.org/2000/svg" width="52" height="64" viewBox="0 0 52 64" class="usa-banner__lock-image" role="img" aria-labelledby="banner-lock-description" focusable="false">
```

Comment:
Just to be sure this used to be described by both the title `banner-lock-title` and `banner-lock-description`. 

Is this change to only have it described by the description intentional?

Reply:
Yes, this was intentional. On VoiceOver, the readout was "A lock ( Lock Locked padlock", so I simplified it by removing the reference to the title. Now that you bring it up, it might be better to just move the description content into the title field and then reference that. Objections?

Reply:
No, just double-checking. Thanks!